lib/android/audio.nit

```diff
@@ -14,11 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Android audio services
+# Android audio services, wrapps a part of
+# http://developer.android.com/reference/android/media/package-summary.html
 #
-# You can get a sound by loading it with a `SoundPool` or a `MediaPlayer`
-# the recommended way to load a sound is by it's resource ID or it's name
-# other ways are for advanced use
+# You should use `load_sound`, `load_music`, `load_sound_from_res`
```

Comment:
You can simplify this doc by cutting the first few words.. `Use...` and on the next sentence `Get a sound ...`